### PR TITLE
Remove cypress and use only playwright for e2e tests

### LINKED ISSUE #659: Change local e2e tests from cypress to playwright
In https://github.com/NASA-IMPACT/admg-casei/issues/457 and related PR https://github.com/NASA-IMPACT/admg-casei/pull/508, the CASEI team replaced cypress with playwright in the github actions suite. However, cypress is still used in the local development environment.

Is there a reason that cypress is still active, or was this simply not removed yet? When e2e tests fail in actions, it would be ideal to be able to debug by testing locally. Consider replacing cypress with playwright for local e2e tests.

### PR DESCRIPTION
**Pull request by willemarcel**
Resolves #659

- Remove package-lock.json and use only yarn in CI
- Use playwright in local tests
- Remove cypress packages
- Update documentation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,10 +108,7 @@
     "test": "jest",
     "tdd": "jest --watch",
     "updateSnapshot": "jest --updateSnapshot",
-    "cy:open": "cypress open",
-    "cy:run": "cypress run",
-    "test:e2e": "start-server-and-test 'yarn develop' http://localhost:8000 'yarn cy:open'",
-    "test:e2e:ci": "start-server-and-test 'yarn serve -p 8000' http://localhost:8000 'yarn cy:run'",
+    "test:e2e": "start-server-and-test 'yarn serve -p 8888' http://localhost:8888/casei 'npx playwright test -c ./playwright/e2e'",
```
Comment (LanesGood) [inline package.json:111]: @willemarcel should this include the build command?

Comment (willemarcel) [reply package.json:111]: As the build takes a couple of minutes, I prefer to execute it separately. Every so often, we don't need a new build, but just to fix a test file and run the test again.

**Review by LanesGood — APPROVED** [review]
Looks ok to me! I ran locally and had a couple failing tests as I believe you did; as you suggested, I think this may be from slower API calls from running on a local build.

**Comment by LanesGood** [comment]
Also nice removal of packagelock.json! Good to get rid of that warning

**Comment by willemarcel** [comment]
> Looks ok to me! I ran locally and had a couple failing tests as I believe you did; as you suggested, I think this may be from slower API calls from running on a local build.

I think the problem is that the build command doesn't read the mapbox token from the `.env.development` file. You need to set the env var in the terminal before running the command. This solved the issue I was experiencing.

**Pull request merged**